src/core/dev_api/openvino/runtime/itensor.hpp

```diff
@@ -0,0 +1,135 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include "openvino/core/coordinate.hpp"
+#include "openvino/core/core_visibility.hpp"
+#include "openvino/core/shape.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "openvino/runtime/allocator.hpp"
+
+namespace InferenceEngine {
+
+class Blob;
+
+}
+
+namespace ov {
+
+class OPENVINO_API ITensor : public std::enable_shared_from_this<ITensor> {
```

Comment:
why do we need dev api for tensor?

Reply:
We don't need this, I had a plan to remove it from developer API